moped-editor/src/components/GridTable/GridTableSearch.js

```diff
@@ -35,14 +32,48 @@ const useStyles = makeStyles(theme => ({
     },
   },
   downloadButtonGrid: {
-    padding: "1rem 1rem 0 1rem",
+    padding: "12px",
+    [theme.breakpoints.between("sm", "md")]: {
+      padding: 0,
+    },
   },
   downloadCsvButton: {
     height: "2.5rem",
   },
   tabStyle: {
     margin: ".5rem",
   },
+  searchBarContainer: {
+    padding: "24px",
+    [theme.breakpoints.down("xs")]: {
+      paddingBottom: "12px",
+    },
+  },
+  advancedSearchRoot: {
+    width: "100%",
+    zIndex: "3",
+    paddingLeft: "43px",
+    paddingRight: "17px",
+    [theme.breakpoints.down("sm")]: {
+      paddingLeft: "23px",
+      paddingRight: "14px",
```

Comment:
I could not figure out what the formula is for this padding. Basically setting the Popper to 100% width made it even wider than the screen. I can't define the width of the element in pixels because then it wouldnt be reponsive. These numbers feel random, but they work. Yes it's weird.